### PR TITLE
Added the basic sections in My Portfolio page and made UI enhancements.

### PR DESCRIPTION
**Pull request by TanyaChauhan9**

### CODE REVIEW

File: portfolio/src/main/webapp/images/linkedin.svg [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		c87.984-14.78,159.5-77.993,186.175-161.311c0.849-2.716,1.658-5.452,2.432-8.206C465.082,301.055,465.09,301.032,465.097,301.017z
+		"/>
+</g>
+<path style="fill:#FFFFFF;" d="M358.565,230.459v87.883h-50.944v-81.997c0-20.595-7.375-34.656-25.811-34.656
+	c-14.084,0-22.458,9.474-26.147,18.634c-1.343,3.278-1.688,7.835-1.688,12.423v85.593H203.02c0,0,0.681-138.875,0-153.259h50.952
+	V186.8c-0.094,0.161-0.236,0.34-0.329,0.498h0.329V186.8c6.769-10.425,18.862-25.324,45.923-25.324
+	C333.432,161.479,358.565,183.384,358.565,230.459z M149.7,91.198c-17.429,0-28.838,11.439-28.838,26.473
+	c0,14.716,11.072,26.495,28.164,26.495h0.344c17.766,0,28.823-11.779,28.823-26.495C177.857,102.636,167.137,91.198,149.7,91.198z
+	 M123.886,318.341h50.944V165.083h-50.944V318.341z"/>
+<g>
```
Comment (rahulnagurtha) [inline portfolio/src/main/webapp/images/linkedin.svg:23]: What are these characters ?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,243 @@
...
+                    </a>
+                </div>
+            </div><!--//profile-->              
+        </div><!--//container-->
+        <div class="content">
+            <div class="leftSide">
+                <div class="Aboutme">
+                    <h2>About me</h2>
+                    <p>Hi! I'm Tanya. 
+                    I'm pursuing my Bachelor's in Mathematics and Computing from Indian Institure of Techology, Guwahati.
```
Comment (rahulnagurtha) [inline portfolio/src/main/webapp/index.html:33]: Institute

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,243 @@
...
+                </div>
+            </div><!--//profile-->              
+        </div><!--//container-->
+        <div class="content">
+            <div class="leftSide">
+                <div class="Aboutme">
+                    <h2>About me</h2>
+                    <p>Hi! I'm Tanya. 
+                    I'm pursuing my Bachelor's in Mathematics and Computing from Indian Institure of Techology, Guwahati. 
+                    Things that interest me the most are music, coding, working out, novels, astrophysics and a starry night sky!
```
Comment (rahulnagurtha) [inline portfolio/src/main/webapp/index.html:34]: Starry night sky..... I could get a glimpse of stars at night time when I was in Guwahati but in Hyderabad I rarely find them. I guess the same would be the case for most of the metros.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,18 +1,243 @@
...
+            <div class="leftSide">
+                <div class="Aboutme">
+                    <h2>About me</h2>
+                    <p>Hi! I'm Tanya. 
+                    I'm pursuing my Bachelor's in Mathematics and Computing from Indian Institure of Techology, Guwahati. 
+                    Things that interest me the most are music, coding, working out, novels, astrophysics and a starry night sky! 
+                    Also, I love having conversation with all kinds of people and gain experience from them.
+                    <br/>
+                    Recently, I've developed an interest for coding and both the thrill of speed and the patience required for diving deep into problems excite me!
+                    I've also worked on various software projects both during my college and my internships and I've worked hard to learn to write good quality codes.
```
Comment (rahulnagurtha) [inline portfolio/src/main/webapp/index.html:38]: nit: *good quality code

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,18 +1,243 @@
...
+                        </p>
+                    </div>
+                    <br />
+                    <br />
+                    <img class="projectLogo" src="../images/msLogo.jpg">
+                    <div class="projectDesc">
+                        <h3>Modern Employee Intranet App</h3>
+                        <p>Developed a Modern Employee Intranet app on Microsoft Teams, to host all features and needs
+                            for any organisation on a single platform. Implemented command-based intelligent bots to identify, understand and proactively reply using rich cards, message extension to query through content, and task modules to create a modal popup experience.
+                            The app was created on C# and .NET, used cognitive services like LUIS to develop an NLP based bot.
```
Comment (rahulnagurtha) [inline portfolio/src/main/webapp/index.html:64]: Suggestion: When you are talking about not so common acronyms like LUIS, it's good to hyperlink it.

**Review by rahulnagurtha — APPROVED** [review]
Impressive achievements and credentials :)

File: portfolio/src/main/webapp/images/linkedin.svg (hunk 1 above)
Comment (TanyaChauhan9) [reply portfolio/src/main/webapp/images/linkedin.svg:23]: These characters are present in all the svg type files. These attributes define the shape, style etc of the image.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (TanyaChauhan9) [reply portfolio/src/main/webapp/index.html:34]: Yes. Same for Delhi as well :/

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (TanyaChauhan9) [reply portfolio/src/main/webapp/index.html:64]: Noted! Will link this.

**Pull request merged**